### ISSUE TITLE
Need new test: Protocols/Technology - LACP: Add/remove links physically

### ISSUE THREAD

**Issue by kevinskywang** [Azure/sonic-mgmt#3607]
This is a test gap:

Need to test LAG physical failure detection.

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by kamalsahu0001** [comment]
Hi @yxieca,

For this test the hard requirement is to physically remove the cable or if we can simulate cable disconnect, it will suffice the requirement?

**Comment by yxieca** [comment]
I don't think we can automate cable remove. Turning off laser on the remote side is the best we can do for automation.

**Comment by kamalsahu0001** [comment]
Keysight will be taking care of this test case. We will cover it by simulating link down on test port side and will verify the convergence time.

**Comment by vaibhavhd** [comment]
@kamalsahu0001 do you want to address this testgap as past of snappy (where we have IXIA) testcases?
